docs/assistant/cloud/collections.md

```diff
@@ -0,0 +1,75 @@
+---
+id: collections
+title: Collections
+sidebar_label: Collections
+---
+
+The Collection feature lets you organize your library assets similar to how WordPress post categories work.
+
+## Assign to multiple Collections
+
+Any creative asset (posts, pages, images, etc.) can be assigned to more than one collection, and it will appear under each collection. Organizing files this way is preferable to creating folders, because files can be referred to multiple places.
+
+In the image below, the color `#ede2f1` is filed under two collections; **Pastel Colors** and **Pink Colors**.
+
+![Organizing colors](/img/assistant/cloud--libraries--organize-assets--1.jpg)
+
+## Create a Collection from the Assistant Pro Cloud
+
+To create a library collection from the Assistant Pro Cloud:
+
+1. Select the library you’d like to add a collection.
+2. Select the **Collections** tab
+3. Type a name for your collection
+4. Click the **Add Collection** button
+
+![Create collection from Assistant Pro Cloud](/img/assistant/cloud--libraries--organize-assets--2.jpg)
+
+## Create a Library Collection from the Assistant Plugin
+
+To create a library collection from the Assistant Plugin:
+
+1. Click on the library you’d like to add a collection.
+2. Click the **Settings** icon.
+3. Type a name for your collection
+4. Click the **Add Collection** button
+
+![Create collection from Assistant plugin](/img/assistant/cloud--libraries--organize-assets--3.jpg)
+
+## Assign Collections to a creative asset from Assistant Pro Cloud
+
+To add collections to a creative asset:
+
+1. Select the asset.
+2. Enter collection names into the sidebar dialogue field. 
+
+![Assign assets to collections from Assistant Pro Cloud](/img/assistant/cloud--libraries--organize-assets--4.jpg)
+
+## Assign Collections to a creative asset from the Assistant Plugin
```

Comment:
If you agree this procedure is identical to the previous one, you can delete lines 48-53.